src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,154 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+import mate.academy.internetshop.annotations.Dao;
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.model.Item;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractDao<Item> implements ItemDao {
+    private static final Logger logger = Logger.getLogger(ItemDaoJdbcImpl.class);
+    private static final String DB_NAME = "dbinternetshop";
+
+    public ItemDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public Item create(Item item) {
+        Statement stmt = null;
+        String query = "Insert into " + DB_NAME + ".items (name, price) Values ("
```

Comment:
try to use String.format() in all queries

Reply:
OK.